azure-pipelines.yml

```diff
@@ -32,6 +32,8 @@ jobs:
   steps:
   - script: git submodule update --init --recursive
     displayName: 'Checkout Submodules'
+  - script: sudo xcode-select -s "/Applications/Xcode_12.2.app"
```

Comment:
I couldn't track down a date for when Azure is switching to Xcode 12.2 by default.